.github/actionlint.yaml

```
@@ -0,0 +1,5 @@
+self-hosted-runner:
```

Fixes the `yamllint` error:

```suggestion
---
self-hosted-runner:
```